commander/src/commands/transaction/create/vote.ts

```diff
@@ -157,7 +157,7 @@ export default class VoteCommand extends BaseCommand {
 
 			return {
 				delegateAddress,
-				amount: String(Number(normalizedAmount) * sign),
+				amount: String(BigInt(normalizedAmount) * BigInt(sign)),
```

Comment:
```suggestion
				amount: (BigInt(normalizedAmount) * BigInt(sign)).toString(),
```

Reply:
I think both work

Reply:
yes, `String` constructor also works, but we don't use it anywhere else, so let's keep it consistent